app/renderer/components/Inspector/LocatedElements.js

```diff
@@ -31,60 +34,65 @@ class LocatedElements extends Component {
-            >
-              {t('Copy ID')}
-            </Button>
-          </div>
-          <div>
-            <Button size='small'
-              disabled={!locatorTestElement}
-              onClick={() => applyClientMethod({methodName: 'click', elementId: locatorTestElement})}
-            >
-              {t('Tap Element')}
-            </Button>
-          </div>
-          <div>
-            <Button size='small'
-              disabled={!locatorTestElement}
-              onClick={() => applyClientMethod({methodName: 'clear', elementId: locatorTestElement})}
-            >
-              {t('Clear')}
-            </Button>
-          </div>
-          <div className={InspectorStyles['send-keys-container']}>
-            <Input size='small' placeholder={t('Enter keys')} onChange={(e) => this.setState({...this.state, sendKeys: e.target.value})}/>
-            <Button size='small'
-              disabled={!locatorTestElement}
-              onClick={() => applyClientMethod({methodName: 'sendKeys', elementId: locatorTestElement, args: [this.state.sendKeys || '']})}
-            >
-              {t('Send Keys')}
-            </Button>
-          </div>
-        </div>}
-      </Col>
-    </Row>;
+    return <>
+      {locatedElements.length === 0 && <Row><i>{t('couldNotFindAnyElements')}</i></Row>}
+      {locatedElements.length > 0 && <Space className={InspectorStyles.spaceContainer} direction='vertical' size='small'>
+        <Row><span>{t('elementsCount')} <Badge count={locatedElements.length} offset={[0, -2]}/></span></Row>
+        <Row>
+          <List className={InspectorStyles.searchResultsList}
+            size='small'
+            dataSource={locatedElements}
+            renderItem={(elementId) =>
+              <List.Item type='text'
+                className={locatorTestElement === elementId ? InspectorStyles.searchResultsSelectedItem : ''}
+                onClick={() => setLocatorTestElement(elementId)}
+              >
+                {elementId}
+              </List.Item>
+            }
+          />
+        </Row>
+        <Row justify='center'>
+          <Space direction='horizontal' size='small'>
+            <Tooltip title={t('Copy ID')} placement='bottom'>
+              <Button
+                disabled={!locatorTestElement}
+                icon={<CopyOutlined/>}
+                onClick={() => clipboard.writeText(locatorTestElement)}/>
+            </Tooltip>
+            <Tooltip title={t('Tap')} placement='bottom'>
+              <Button
+                disabled={!locatorTestElement}
+                icon={<AimOutlined/>}
+                onClick={() => applyClientMethod({methodName: 'click', elementId: locatorTestElement})}
+              />
+            </Tooltip>
+            <ButtonGroup className={InspectorStyles.searchResultsActions}>
+              <Input className={InspectorStyles.searchResultsKeyInput}
+                disabled={!locatorTestElement}
+                placeholder={t('Enter keys')}
```

Comment:
maybe something like 'Keys to send' would be slightly more understandable?

Reply:
Changed to 'Enter Keys to Send'